recommender-back/src/apis/forecast.py

```diff
@@ -17,21 +17,26 @@ def update_data(self):
         current, start, end = get_forecast_times()
         bbox = '24.5,60,25.5,60.5'
         timestep = 60
+        parameters = 'Temperature,Humidity,WindUMS,WindVMS,PrecipitationAmount,TotalCloudCover'
```

Comment:
Added params so downloaded file size is smaller (makes no difference in download time but makes parsing more efficient)